config/core/test-configs.yaml

```diff
@@ -88,7 +88,9 @@ file_systems:
     type: debian
     ramdisk: 'buster-v4l2/20211224.3/{arch}/rootfs.cpio.gz'
 
-
+  debian_buster-ltp_ramdisk:
```

Comment:
We've moved on to Bullseye now for this rootfs, so please use `debian_bullseye-ltp_ramdisk` and move it after the `-nfs` variant.

Reply:
Done.